editor/src/clj/editor/collection.clj

```diff
@@ -177,25 +177,46 @@
    :icon ""
    :label ""})
 
-(g/defnk produce-go-build-targets [_node-id build-error build-resource ddf-message resource-property-build-targets source-build-targets pose]
-  ;; Create a build-target for the referenced or embedded game object. Also tag
-  ;; on :game-object-instance-data with the overrides for this instance. This
-  ;; will later be extracted and compiled into the Collection - the overrides do
-  ;; not end up in the resulting game object binary.
```

Comment:
Split into `produce-referenced-go-build-targets` and `produce-embedded-go-build-targets`.